game/game_controller.rb

```diff
@@ -0,0 +1,31 @@
+require_relative 'game'
+require_relative 'author'
+
+class GameController
+  attr_accessor :games, :authors
+
+  def initialize
+    @games = []
+    @authors = []
+  end
+
+  def list_games
+    @games.map { |game| puts "Multiplayer: #{game.multiplayer}, Last played at: #{game.last_played_at}" }
+  end
+
+  def list_authors
+    @authors.map { |author| puts "Name: #{author.first_name author.last_name}" }
+  end
```

Comment:
- [ ] When I try listing all authors, I see that it does nothing. Listing the games and adding the games all work as expected. Kindly include the add authors method to make this part complete. Or if it is addressed in a separate milestone you can ignore this.

Reply:
Thank you. I noticed it and addressed it in another milestone. It works as expected now.